docs/release-notes/2.9.1.md

```diff
@@ -0,0 +1,55 @@
+# Percona Monitoring and Management 2.9.1
+
+<table class="docutils field-list" frame="void" rules="none">
+  <colgroup>
+    <col class="field-name">
+    <col class="field-body">
+  </colgroup>
+  <tbody valign="top">
+    <tr class="field-odd field">
+      <th class="field-name">Date:</th>
+      <td class="field-body">August 3, 2020</td>
+    </tr>
+    <tr class="field-even field">
+      <th class="field-name">Installation:</th>
+      <td class="field-body">
+        <a class="reference external" href="https://www.percona.com/doc/percona-monitoring-and-management/2.x/install/index-server.html">Installing Percona Monitoring and Management</a></td>
+    </tr>
+  </tbody>
+</table>
+
+*PMM* ([Percona Monitoring and Management](https://www.percona.com/doc/percona-monitoring-and-management/index.html))
+is a free and open-source platform for managing and monitoring MySQL, MongoDB, and PostgreSQL
+performance.
+
+## Improvements
+
+* [PMM-6230](https://jira.percona.com/browse/PMM-6230): Custom dashboards set as Home remain so after update
+* [PMM-6300](https://jira.percona.com/browse/PMM-6300): Query Analytics Dashboard: Column sorting arrows made easier to use (Thanks to user debug for reporting this issue)
+* [PMM-6208](https://jira.percona.com/browse/PMM-6208): Security Threat Tool: Temporarily silence viewed but unactioned alerts
+* [PMM-6315](https://jira.percona.com/browse/PMM-6315): Query Analytics Dashboard: Improved metrics names and descriptions
+* [PMM-6274](https://jira.percona.com/browse/PMM-6274): MySQL User Details Dashboard: View selected user's queries in Query Analytics Dashboard
+* [PMM-6266](https://jira.percona.com/browse/PMM-6266): Query Analytics Dashboard: Pagination device menu lists 25, 50 or 100 items per page
+* [PMM-6262](https://jira.percona.com/browse/PMM-6262): PostgreSQL Instance Summary Dashboard: Descriptions for all 'Temp Files' views
+* [PMM-6253](https://jira.percona.com/browse/PMM-6253): Query Analytics Dashboard: Improved SQL formatting in Examples panel
+* [PMM-6211](https://jira.percona.com/browse/PMM-6211): Query Analytics Dashboard: Loading activity spinner for Example, Explain and Tables tabs
```

Comment:
not sure I understand what this is...was it added, improved, changed, removed?

Reply:
It's new. I'll make that clearer.